### PR TITLE
[16.01] Fix common_startup options for planemo.

### PR DESCRIPTION
**Pull request by jmchilton**
See conversation with myself on https://github.com/galaxyproject/galaxy/pull/1484.

Ping @natefoo.

### CODE REVIEW

File: scripts/common_startup.sh [hunk 1]
```diff
@@ -14,6 +19,7 @@ for arg in "$@"; do
     [ "$arg" = "--skip-venv" ] && SET_VENV=0
```
Comment (natefoo) [inline scripts/common_startup.sh]: This line is no longer necessary.

**Comment by natefoo** [comment]
Seems good to me, the test errors are unrelated?

**Comment by martenson** [comment]
@natefoo yes, they are both timeouts

**Comment by natefoo** [comment]
+1

**Comment by jmchilton** [comment]
@natefoo I fixed the left over extra check of skip-venv and rebased.

**Pull request merged**